content/en/news/security/istio-security-2020-011/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: ISTIO-SECURITY-2020-011
+subtitle: Security Bulletin
+description: Envoy incorrectly restores the proxy protocol downstream address for non-HTTP connections.
+cves: [N/A]
+cvss: "N/A"
+vector: ""
+releases: ["1.8.0"]
+publishdate: 2020-11-21
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+Envoy, and subsequently Istio, is vulnerable to a newly discovered vulnerability:
+
+- [Incorrect proxy protocol downstream address for non-HTTP connections](https://groups.google.com/g/envoy-security-announce/c/aqtBt5VUor0):
+Envoy incorrectly restores the proxy protocol downstream address for non-HTTP connections. Instead of restoring the address supplied by the proxy protocol filter,
+Envoy restores the address of the directly connected peer and pass it to subsequent filters. This will affect logging (`%DOWNSTREAM_REMOTE_ADDRESS%`) and
```

Comment:
```suggestion
Envoy restores the address of the directly connected peer and passes it to subsequent filters. This will affect logging (`%DOWNSTREAM_REMOTE_ADDRESS%`) and
```